core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlConverter.java

```diff
@@ -132,17 +132,32 @@ public class RelToSqlConverter extends SqlImplementor
   /** Creates a RelToSqlConverter. */
   @SuppressWarnings("argument.type.incompatible")
   public RelToSqlConverter(SqlDialect dialect) {
-    super(dialect);
+    super(dialect, DEFAULT_BLOAT);
     style = new QueryStyle();
     dispatcher = ReflectUtil.createMethodDispatcher(Result.class, this, "visit",
       RelNode.class);
   }
   public RelToSqlConverter(SqlDialect dialect, QueryStyle style) {
-    super(dialect);
+    super(dialect, DEFAULT_BLOAT);
     this.style = style;
     dispatcher = ReflectUtil.createMethodDispatcher(Result.class, this, "visit",
         RelNode.class);
   }
+
+  public RelToSqlConverter(SqlDialect dialect, int bloat) {
+    super(dialect, bloat);
```

Comment:
Instead of `super()`, we can call constructor `RelToSqlConverter(SqlDialect dialect, QueryStyle style, int bloat)`

Reply:
done